packages/connections/src/components/connection-list/connection.tsx

```diff
@@ -57,63 +42,46 @@ const connectionButtonContainerStyles = css({
     '&::after': {
       opacity: 1,
       width: spacing[1],
-      backgroundColor: uiColors.focus,
     },
   },
 });
 
 const connectionButtonStyles = css({
   margin: 0,
-  padding: 0,
-  paddingLeft: spacing[4],
+  padding: `${spacing[1]}px 0 ${spacing[1]}px ${spacing[1] * 3}px`,
```

Comment:
i tried hard to avoid using *3, but could not find a common ground between icon sizes, text alignments. I did however split padding into its corresponding ones.